zerver/tests/test_events.py

```diff
@@ -2633,8 +2633,11 @@ def do_set_realm_property_test(self, name: str) -> None:
                 old_value,
                 val,
             ]:
-                # email update event is sent for each user.
-                num_events = 11
+                # one event for updating "email_address_visibility" setting +
+                # one delivery_email update event is sent for 9 users (all human
+                # users except the user themselves) + one email update event for
+                # each of the 10 human users in the realm.
+                num_events = 20
```

Comment:
I guess this is problematically high (and thus a DoS risk), but that's OK because we're deleting this later in the PR; even the 1 per user was a problem. I found the existing issue for this problem and closed it: https://github.com/zulip/zulip/issues/14600.